object_test.go

```diff
@@ -753,3 +754,77 @@ func TestObjectConvertValueEqual(t *testing.T) {
 	value.chain.assertFailed(t)
 	value.chain.reset()
 }
+
+func TestObjectEvery(t *testing.T) {
```

Comment:
Please also add `Every` invocation to TestObjectFailed and TestArrayFailed.

(pass non-nil func to that invocation to be sure that `Every` handled failed chain, not nil func).